### DISCUSSION TITLE
Android/iOS maps google api pricing once published

### DISCUSSION THREAD

**Discussion by Seandump101** [react-native-maps/react-native-maps#4491] [Q&A]
Can someone please confirm if this actually does use the android and iOS SDK ? 
I have contacted Google and support seems to think that it will be using the JavaScript api and therefore not free.
This is a critical factor many developers need to be 100% clear on before developing a map based app with RN.

**Comment by jan-kozinski** [comment]
Yes it does use Android and IOS SDK
